docs/src/index.md

```diff
@@ -207,24 +219,30 @@ M2 = CirculantMatrix(copy(v));
 Base.sum(M::CirculantMatrix) = length(M.data)*sum(M.data)
 
 d1 = @dspawn begin
+    @W v
+
     sleep(0.5)
-    println("I write to v")
+    println("d1: I write to v")
     fill!(v,0) 
-end (v,) (W,);
+end;
 
 d2 = @dspawn begin
-    println("I wait for d1 to write")
+    @R M1
+
+    println("d2: I wait for d1 to write")
     sum(M1)
-end (M1,) (R,);
+end;
 
-d3= @dspawn begin
-    println("I don't wait for d1 to write")
-    sum(M2)
-end (M2,) (R,);
+d3 = @dspawn begin
+    @R M2
 
-fetch(d2)
+    println("d3: I don't need to wait for d1 to write")
```

Comment:
I don't know why, but this output line never shows up in the documentation.

Reply:
I don't know either... perhaps printing to the repl asynchronously from various tasks is not a very robust thing? In any case, I think using "print" as a way to indicate order was not very good, so I simplified the example a bit.